src/updater.cpp

```diff
@@ -44,6 +44,7 @@ Updater::Updater(QObject* parent, QSettings* settings) : QObject(parent), mUpdat
     this->settings = settings;
 
     feed = new dblsqd::Feed(QStringLiteral("https://feeds.dblsqd.com/MKMMR7HNSP65PquQQbiDIw"), QStringLiteral("release"));
+    installOrRestartButton = new QPushButton(tr("Update"));
```

Comment:
Is `installOrRestartButton` supplied by a `ui_xxxx.h` form/dialog (in which case it should already exist after a call to `setupUi()` and this is a leak of the `QPushButton` instance created by that and thrown away here) or is it a member - which I might have called `mpButton_installOrRestart` and which could either be set to `nullptr` in the initialiser list or possibly the latter could be `, mpButton_installOrRestart(QPushButton(tr("Update")))`?

Reply:
It's a class member pointer - adjusted per your advice.